WhaleSpotting/react-app/src/components/SightingMapInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import WeatherApiModel from "../apiModels/WeatherApiModel";
+import { Chosen } from "./Map";
+import "../styles/SightingMapInfo.scss";
+
+interface infoProps {
+    chosen: Chosen | undefined;
+}
+
+export default function SightingMapInfo({ chosen }: infoProps): JSX.Element {
+
+    const key = process.env.REACT_APP_WEATHER_API_KEY;
+    
+    const response = {
+        lon: chosen?.lon,
+        lat: chosen?.lat,
+        species: ["Whale", "Orca", "Dolphin"]
+    };
+
+    const [weatherData, setWeatherData] = useState<WeatherApiModel>();
+    async function fetchWeather(): Promise<WeatherApiModel | void> {
+        //eslint-disable-next-line
+        return await fetch(`https://api.openweathermap.org/data/2.5/onecall?lat=${chosen!.lat}&lon=${chosen!.lon}&units=metric&Appid=${key}`)
+            .then(response => response.json())
+            .then(response => setWeatherData(response));
+    }
+
+    useEffect(() => {
+        if (chosen) {
+            fetchWeather();
+        }
+    }, [chosen]);
+
+    if (!chosen || !chosen.lat || !chosen.lon || !weatherData) {
```

Comment:
I have a feeling we don't need the lat and lon
```suggestion
    if (!chosen || !weatherData) {
```